mypy/stubgen.py

```diff
@@ -666,8 +670,19 @@ def parse_options(args: List[str]) -> Options:
     search_path = []  # type: List[str]
     interpreter = ''
     include_private = False
+    output_dir = 'out'
     while args and args[0].startswith('-'):
-        if args[0] == '--doc-dir':
+        if args[0] == '--output-dir':
+            output_dir = args[1]
+            args = args[1:]
+            # Resolve/expand output_dir; use symlinks
+            if output_dir.startswith('~'):
+                output_dir = os.path.expanduser(output_dir)
```

Comment:
Normally the shell takes care of this. I think on Windows it may not -- but is it really important enough to support?